### PR TITLE
documentation fixes

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

File: docs/diagnostics/PX1034.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ public class SOOrder : IBqlTable
 
 ## Related Articles
 
-[Foreign Keys and Nullable Columns](https://help.acumatica.com/(W(12))/Help?ScreenId=ShowWiki&pageid=8da9e9c6-ebbf-409a-b43d-a13d2081a62e)
```
Comment (dropsonic) [inline docs/diagnostics/PX1034.md:73]: What has been changed here? Line endings?

**Review by dropsonic — APPROVED** [review]

File: docs/diagnostics/PX1034.md (hunk 1 above)
Comment (EAndrosova) [reply docs/diagnostics/PX1034.md:73]: Yes, I had to change them so that all links are displayed in a list

Comment (SENya1990) [reply docs/diagnostics/PX1034.md:73]: @dropsonic Yes, Github understand two whitespaces as a line break. Try look at the rich diff to see the difference

Comment (SENya1990) [reply docs/diagnostics/PX1034.md:73]: @EAndrosova actually I found that two spaces at the end work as well and the distance between lines is less. This imho makes the list look better

Comment (EAndrosova) [reply docs/diagnostics/PX1034.md:73]: I agree, it seems that only oxygen does not understand white space in markdown

**Review by EAndrosova — APPROVED** [review]

**Pull request merged**